src/EventsPage/components/MyUpcomingEvents.js

```diff
@@ -23,25 +42,28 @@ export default (props) => {
 					</div>
 				</div>
 				<div className="row">
-					{renderEventList(props.myUpcomingEvents)}
+					{sortByDate(renderEventList(props.myUpcomingEvents))}
 				</div>
 			</div>
 		</div>
 	);
 };
 
+
 const renderEventList = (events) => {
-	return events.map(event => {
-		return (
-			<div key={event.id} className="col-xs-offset-1 col-xs-3">
-				<Link to={`/eventsfeed/${event.id}`}>
-					<div className="event">
-						<div className="event-pic"> <img className="img-responsive" src={event.photo} /></div>
-						<div className="event-title"><p>{event.title}</p></div>
-						<div className="event-time"><p>{event.date} <br/> {event.time}</p></div>
-					</div>
-				</Link>
-			</div>
-		);
+	let today = new Date();
+	today = today.toLocaleString('en-US', { year: 'numeric', month: '2-digit', day: '2-digit'})
+		return events.map(event => { if (event.date < today) {
+			return (
+				<div key={event.id} className="col-xs-offset-1 col-xs-3">
+					<Link to={`/eventsfeed/${event.id}`}>
+						<div className="event">
+							<div className="event-pic"> <img className="img-responsive" src={event.photo} /></div>
+							<div className="event-title"><p>{event.title}</p></div>
+							<div className="event-time"><p>{event.date} <br/> {event.time}</p></div>
+						</div>
+					</Link>
+				</div>
+			)};
```

Comment:
This is super-nitpicky, but I'd move the semicolon inside the brace and move the brace to its own line.